manifest.json

```diff
@@ -1,15 +1,22 @@
 {
+  "$schema": "https://json.schemastore.org/chrome-manifest",
   "manifest_version": 3,
   "name": "__MSG_extensionName__",
   "description": "__MSG_extensionDescription__",
   "version": "1.26.0",
   "version_name": "1.26.0-prerelease",
   "default_locale": "en",
+  "minimum_chrome_version": "93",
```

Comment:
We should not enforce a `minimum_chrome_version`.

Reply:
But won't the extension not work then?

Reply:
We already manually enforce it, why not use Chrome's built-in safeguards